src/v3/src/transformer/field/transform.ts

```diff
@@ -13,13 +13,31 @@
 import { IdxTransaction, Input, NextStep } from '@okta/okta-auth-js';
 
 import {
-  FieldElement, FormBag, StepTransformer, UISchemaElement,
+  FieldElement, FormBag, StepTransformer, TranslationInfo, UISchemaElement,
 } from '../../types';
-import { flattenInputs } from '../../util';
+import { flattenInputs, loc } from '../../util';
 import { createForm } from '../utils';
 import { transformer as attributesTransformer } from './attributes';
 import { transformer as typeTransformer } from './type';
 
+const getAdditionalUITranslations = (name: string | undefined): TranslationInfo[] | undefined => {
```

Comment:
how about just make another step in transformer/authJs.ts, like transformI18n ?

Reply:
Done. 🙏